docs/docs/guides/integrate/identity-providers/linkedin_oauth.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Configure LinkedIn as an OAuth Identity Provider in ZITADEL
+sidebar_label: LinkedIn generic OIDC
+id: linkedin-oauth
+---
+
+import GeneralConfigDescription from './_general_config_description.mdx';
+import Intro from './_intro.mdx';
+import CustomLoginPolicy from './_custom_login_policy.mdx';
+import IDPsOverview from './_idps_overview.mdx';
+import GenericOAuth from './_generic_oauth.mdx';
+import Activate from './_activate.mdx';
+import PrefillAction from './_prefill_action.mdx';
+import TestSetup from './_test_setup.mdx';
+
+<Intro provider="LinkedIn"/>
+
+## LinkedIn Configuration
+
+### Register a new client
+
+1. Go to the LinkedIn Developer console and create a new App: https://www.linkedin.com/developers/apps/new
+2. Add your App Name, your Company Page and a Logo
+3. Add "Sign In with LinkedIn using OpenID Connect" by clicking "Request access"
+4. Go to the Auth Settings of the App and add the following URL to the "Authorized redirect URLs"
+ - {your-domain}/ui/login/login/externalidp/callback
+ - Example redirect url for the domain `https://acme.zitadel.cloud` would look like this:  `https://acme.zitadel.cloud/ui/login/login/externalidp/callback`
+5. Verify the app as your company
```

Comment:
I wasn't asked for any verification for my company app. Is that because I'm already the admin?

Reply:
Not sure, I quickly read of on this, but couldn't find anything about it. Everywhere I search I only find the process with the verification.
But could be because you are the only one of your organization?